common/src/IO/LoadTextureCollection.cpp

```diff
@@ -48,6 +48,8 @@
 #include <kdl/string_format.h>
 #include <kdl/vector_utils.h>
 
+#include <FreeImage.h>
```

Comment:
We currently only have a freeimage dependency in "ReadFreeImageTexture.cpp", and I'd like to keep it that way. I would suggest to add expose a new function such as `isSupportedFreeImageExtension` or something with a better name that checks a given extension.

Reply:
That's a wise rule. I'll keep it in mind next time.